Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift

```diff
@@ -64,9 +64,11 @@ extension PhotonActionSheetProtocol {
             
             let controller = ThemedNavigationController(rootViewController: settingsTableViewController)
             // On iPhone iOS13 the WKWebview crashes while presenting file picker if its not full screen. Ref #6232
-            if UIDevice.current.userInterfaceIdiom == .phone {
+            // Fix for versions less than 14.0
+            if #available(iOS 14.0, *) {} else if UIDevice.current.userInterfaceIdiom == .phone {
                 controller.modalPresentationStyle = .fullScreen
             }
+            }
```

Comment:
Oh no! In my suggestion there was an extra `}` 😞 Do you mind removing it @junbio ?